charts/prefect-agent/values.yaml

```
@@ -65,6 +65,8 @@ agent:
     # -- prefect API url
     apiUrl: http://127.0.0.1:4200/api
 
+  # -- enable this to grant additional permission to the agent role if you wish to use the Dask task runner
```

I'm hesitant because this is such a specific use case, but I guess the cost is low and people using Prefect on K8s are the ones likely to also use Dask on K8s 🤷‍♀️ 
```suggestion
  # -- enable this to grant additional permission to the agent role if you are using the Dask task runner to create dynamic Dask Kubernetes pods 
```